cluster-provision/k8s/check-cluster-up.sh

```diff
@@ -111,7 +116,15 @@ export KUBEVIRTCI_GOCLI_CONTAINER=quay.io/kubevirtci/gocli:latest
 
         echo "Sanity check cluster-up of single stack cluster"
         make cluster-down
+        export KUBEVIRT_WITH_CNAO=false
+        export KUBEVIRT_WITH_MULTUS_V3=false
+        export KUBEVIRT_DEPLOY_ISTIO=false
+        export KUBEVIRT_DEPLOY_PROMETHEUS=false
+        export KUBEVIRT_DEPLOY_PROMETHEUS_ALERTMANAGER=false
+        export KUBEVIRT_DEPLOY_GRAFANA=false
         export KUBEVIRT_SINGLE_STACK=true
+        export KUBEVIRT_DEPLOY_CDI=false
+        unset KUBEVIRT_STORAGE
```

Comment:
What’s the reason for unsetting KUBEVIRT_STORAGE?

Reply:
The latest sanity check should be sized down, therefore I wanted to avoid installing rook-ceph, and thus I unset the value